### ISSUE TITLE
Custom date setting in Analytics has style issue

### ISSUE THREAD

**Issue by 1000camels** [woothemes/woocommerce#35776]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

There seems to be a style issue with the days of the week and the monthly arrows for Custom date setting under Analytics. The arrows are large and the days of the week are not aligned horizontally.

### Expected behavior

I would expect a small arrow and the days of the week to be horizontal

### Actual behavior

The arrows are large and not very usable

### Steps to reproduce

Try to set a custom date in Analytics

### WordPress Environment

`
### WordPress Environment ###

WordPress address (URL): https://cheeseclub
Site address (URL): https://cheeseclub
WC Version: 7.1.0
REST API Version: ✔ 7.1.0
WC Blocks Version: ✔ 8.7.5
Action Scheduler Version: ✔ 3.4.0
Log Directory Writable: ✔
WP Version: 6.1.1
WP Multisite: –
WP Memory Limit: 1 GB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache/2.4.46 (Unix) mod_fastcgi/mod_fastcgi-SNAP-0910052141 OpenSSL/1.0.2u mod_wsgi/3.5 Python/2.7.13
PHP Version: 8.0.8
PHP Post Max Size: 8 MB
PHP Time Limit: 600
PHP Max Input Vars: 1000
cURL Version: 7.68.0
OpenSSL/1.0.2u

SUHOSIN Installed: –
MySQL Version: 5.7.34-log
Max Upload Size: 8 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 7.1.0
WC Database Prefix: wp_
Total Database Size: 3917.98MB
Database Data Size: 3109.28MB
Database Index Size: 808.70MB
wp_woocommerce_sessions: Data: 11.31MB + Index: 0.11MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 32.56MB + Index: 13.55MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 319.84MB + Index: 325.94MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 13.52MB + Index: 15.13MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 11.52MB + Index: 10.03MB + Engine InnoDB
wp_ac_segments: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_admin_columns: Data: 0.11MB + Index: 0.02MB + Engine InnoDB
wp_aioseo_cache: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_aioseo_notifications: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_aioseo_posts: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_berocket_termmeta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_check_email_log: Data: 1715.59MB + Index: 0.00MB + Engine InnoDB
wp_commentmeta: Data: 0.09MB + Index: 0.11MB + Engine InnoDB
wp_comments: Data: 90.61MB + Index: 37.09MB + Engine InnoDB
wp_email_log: Data: 1.52MB + Index: 0.00MB + Engine InnoDB
wp_e_events: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_e_notes: Data: 0.02MB + Index: 0.17MB + Engine InnoDB
wp_e_notes_users_relations: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_e_submissions: Data: 4.52MB + Index: 9.11MB + Engine InnoDB
wp_e_submissions_actions_log: Data: 1.52MB + Index: 1.50MB + Engine InnoDB
wp_e_submissions_values: Data: 3.52MB + Index: 3.03MB + Engine InnoDB
wp_gglcptch_allowlist: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_gglcptch_whitelist: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_hk_addresses: Data: 0.09MB + Index: 0.00MB + Engine InnoDB
wp_icl_content_status: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_icl_core_status: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_icl_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_icl_languages: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_icl_languages_translations: Data: 0.19MB + Index: 0.09MB + Engine InnoDB
wp_icl_locale_map: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_icl_message_status: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_icl_mo_files_domains: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_icl_node: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_icl_reminders: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_icl_strings: Data: 0.30MB + Index: 0.31MB + Engine InnoDB
wp_icl_string_batches: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_icl_string_packages: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_icl_string_positions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_icl_string_status: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_icl_string_translations: Data: 0.19MB + Index: 0.03MB + Engine InnoDB
wp_icl_translate: Data: 4.52MB + Index: 0.08MB + Engine InnoDB
wp_icl_translate_job: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_icl_translations: Data: 0.05MB + Index: 0.06MB + Engine InnoDB
wp_icl_translation_batches: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_icl_translation_downloads: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_icl_translation_status: Data: 1.52MB + Index: 0.02MB + Engine InnoDB
wp_jet_post_types: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_jet_reviews: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_jet_review_comments: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_jet_review_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_jet_review_types: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_jet_taxonomies: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_jet_widgets_styles: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 9.45MB + Index: 0.09MB + Engine InnoDB
wp_pmxe_exports: Data: 5.52MB + Index: 0.00MB + Engine InnoDB
wp_pmxe_google_cats: Data: 0.39MB + Index: 0.00MB + Engine InnoDB
wp_pmxe_posts: Data: 5.52MB + Index: 0.00MB + Engine InnoDB
wp_pmxe_templates: Data: 0.27MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_files: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_hash: Data: 0.09MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_history: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_images: Data: 0.19MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_imports: Data: 0.47MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_posts: Data: 6.52MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_templates: Data: 0.33MB + Index: 0.00MB + Engine InnoDB
wp_postmeta: Data: 630.84MB + Index: 229.66MB + Engine InnoDB
wp_posts: Data: 86.58MB + Index: 13.06MB + Engine InnoDB
wp_redirection_404: Data: 0.09MB + Index: 0.05MB + Engine InnoDB
wp_redirection_groups: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_redirection_items: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_redirection_logs: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_referral_program: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_relevanssi: Data: 1.52MB + Index: 2.39MB + Engine InnoDB
wp_relevanssi_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_relevanssi_stopwords: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_say_what_strings: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_smush_dir_images: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_stream: Data: 4.52MB + Index: 7.92MB + Engine InnoDB
wp_stream_meta: Data: 9.52MB + Index: 15.55MB + Engine InnoDB
wp_termmeta: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 1.52MB + Index: 0.52MB + Engine InnoDB
wp_term_taxonomy: Data: 0.08MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 45.58MB + Index: 40.14MB + Engine InnoDB
wp_users: Data: 2.52MB + Index: 2.55MB + Engine InnoDB
wp_wcpdf_invoice_number: Data: 1.52MB + Index: 0.00MB + Engine InnoDB
wp_wcpdf_packing_slip_number: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 1.52MB + Index: 2.02MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 1.52MB + Index: 0.59MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 38.58MB + Index: 48.13MB + Engine InnoDB
wp_wc_order_stats: Data: 4.52MB + Index: 4.55MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_points_rewards_user_points: Data: 2.52MB + Index: 4.03MB + Engine InnoDB
wp_wc_points_rewards_user_points_log: Data: 5.52MB + Index: 6.55MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.20MB + Index: 0.20MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 1.52MB + Index: 1.22MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.13MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wdr_order_discounts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wdr_order_item_discounts: Data: 0.06MB + Index: 0.05MB + Engine InnoDB
wp_wdr_rules: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfblockediplog: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfblocks7: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wfconfig: Data: 0.48MB + Index: 0.00MB + Engine InnoDB
wp_wfcrawlers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wffilechanges: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wffilemods: Data: 6.52MB + Index: 0.00MB + Engine InnoDB
wp_wfhits: Data: 0.11MB + Index: 0.05MB + Engine InnoDB
wp_wfhoover: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wfissues: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wfknownfilelist: Data: 3.52MB + Index: 0.00MB + Engine InnoDB
wp_wflivetraffichuman: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wflocs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wflogins: Data: 0.17MB + Index: 0.06MB + Engine InnoDB
wp_wfls_2fa_secrets: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wfls_settings: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfnotifications: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfpendingissues: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wfreversecache: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfsnipcache: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wfstatus: Data: 0.13MB + Index: 0.09MB + Engine InnoDB
wp_wftrafficrates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woo_shippment_provider: Data: 0.08MB + Index: 0.00MB + Engine InnoDB
wp_wpda_csv_uploads: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_csv_uploads_BACKUP_20221014010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_csv_uploads_BACKUP_20221104010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_csv_uploads_BACKUP_20221106212953: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_csv_uploads_BACKUP_20221124010044: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_csv_uploads_BACKUP_20221124015504: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_logging: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_logging_BACKUP_20221014010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_logging_BACKUP_20221104010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_logging_BACKUP_20221106212953: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_logging_BACKUP_20221124010044: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_logging_BACKUP_20221124015504: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_media: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_media_BACKUP_20221014010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_media_BACKUP_20221104010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_media_BACKUP_20221106212953: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_media_BACKUP_20221124010044: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_media_BACKUP_20221124015504: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_menus: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_menus_BACKUP_20221014010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_menus_BACKUP_20221104010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_menus_BACKUP_20221106212953: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_menus_BACKUP_20221124010044: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_menus_BACKUP_20221124015504: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_project: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wpda_project_BACKUP_20221014010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_project_BACKUP_20221104010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_project_BACKUP_20221106212953: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_project_BACKUP_20221124010044: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_project_BACKUP_20221124015504: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_project_page: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wpda_project_page_BACKUP_20221014010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_project_page_BACKUP_20221104010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_project_page_BACKUP_20221106212953: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_project_page_BACKUP_20221124010044: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_project_page_BACKUP_20221124015504: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_project_table: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_project_table_BACKUP_20221014010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_project_table_BACKUP_20221104010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_project_table_BACKUP_20221106212953: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_project_table_BACKUP_20221124010044: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_project_table_BACKUP_20221124015504: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_publisher: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wpda_publisher_BACKUP_20221014010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_publisher_BACKUP_20221104010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_publisher_BACKUP_20221106212953: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_publisher_BACKUP_20221124010044: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_publisher_BACKUP_20221124015504: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_table_design: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_table_design_BACKUP_20221014010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_table_design_BACKUP_20221104010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_table_design_BACKUP_20221106212953: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_table_design_BACKUP_20221124010044: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_table_design_BACKUP_20221124015504: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_table_settings: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_table_settings_BACKUP_20221014010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_table_settings_BACKUP_20221104010025: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_table_settings_BACKUP_20221106212953: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_table_settings_BACKUP_20221124010044: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpda_table_settings_BACKUP_20221124015504: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpforms_tasks_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpfront_ure_login_redirect: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wpfront_ure_options: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wpfront_ure_post_type: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wpfront_ure_taxonomy: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wpmailsmtp_tasks_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpml_mails: Data: 1.52MB + Index: 0.00MB + Engine InnoDB
wp_wpr_rocket_cache: Data: 0.20MB + Index: 0.23MB + Engine InnoDB
wp_wpr_rucss_used_css: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_yoast_indexable: Data: 10.52MB + Index: 10.09MB + Engine InnoDB
wp_yoast_indexable_hierarchy: Data: 0.30MB + Index: 0.45MB + Engine InnoDB
wp_yoast_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_yoast_primary_term: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_yoast_seo_links: Data: 0.31MB + Index: 0.25MB + Engine InnoDB

### Post Type Counts ###

acf-field: 127
acf-field-group: 17
attachment: 2465
custom_css: 3
elementor_font: 2
elementor_icons: 2
elementor_library: 105
et_pb_layout: 3
gr_redirect: 1
jet-menu: 1
jet-smart-filters: 6
jet-woo-builder: 1
mailpoet_page: 1
mc4wp-form: 1
monthly_club: 24
nav_menu_item: 10
oembed_cache: 57
page: 51
partner: 102
post: 11
postman_sent_mail: 1
product: 1378
product_icon: 93
product_variation: 2975
revision: 11133
shop_coupon: 3238
shop_order: 37221
shop_order_refund: 833
tablepress_table: 1
wp_global_styles: 1
wpcode: 4

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (1) ###

WooCommerce: by Automattic – 7.1.0

### Inactive Plugins (75) ###

Admin Columns Pro: by AdminColumns.com – 5.7.4
Admin Columns Pro - Advanced Custom Fields (ACF): by AdminColumns.com – 3.0.4
Admin Columns Pro - Cheese Club: by Darcy Christ <darcy@aporia.info> – 1.1
Admin Columns Pro - WooCommerce: by AdminColumns.com – 3.7.3
Admin Columns Pro - Yoast SEO: by AdminColumns.com – 1.2
Advanced Custom Fields PRO: by WP Engine – 6.0.5
Advanced Shipment Tracking for WooCommerce: by zorem – 3.5.1
Category Order and Taxonomy Terms Order: by Nsp-Code – 1.7.4
Check & Log Email: by WPChill – 1.0.6
Cheese Club Extensions: by Darcy Christ <darcy@aporia.info> – 2.1.2
Content Mask: by Alex Demchak – 1.8.4.14
Cookie Notice & Compliance for GDPR / CCPA: by Hu-manity.co – 2.4.2
Custom Post Type UI: by WebDevStudios – 1.13.1
Disable Emails: by WebAware – 1.8.1
Disable jQuery Migrate: by Ivan Blagdan – 1.0
Display Environment Type: by Roy Tanck – 1.3.1
Divit Rewards and Payment Gateway: by Divit team – 1.4.8
Duplicate Page: by mndpsingh287 – 4.4.9
DynamicConditions: by RTO GmbH – 1.6.0
Ele Custom Skin: by Dudaster.com – 3.1.7
Ele Custom Skin Pro: by Dudaster.com – 3.2.4
Elementor: by Elementor.com – 3.8.1
Elementor Pro: by Elementor.com – 3.8.2
Email Log: by Sudar – 2.4.8
Enable Media Replace: by ShortPixel – 4.0.1
Mailgun: by Mailgun – 1.8.7
Members: by MemberPress – 3.2.2
Nav Menu Roles: by Kathy Darling – 2.1.0
PDF Invoices & Packing Slips for WooCommerce: by WP Overnight – 3.2.5
Post Types Order: by Nsp Code – 1.9.9.2
Preview E-mails for WooCommerce: by Digamber Pradhan – 2.1.1
Query Monitor: by John Blackbourn – 3.10.1
Quick Featured Images: by Kybernetik Services – 13.6.0
Redirection: by John Godley – 5.3.5
Redis Object Cache: by Till Krüss – 2.2.2
Relevanssi: by Mikko Saari – 4.18.0
Reorder Posts: by Ryan Hellyer
Ronald Huereca
Scott Basgaard – 2.5.3

Rewrite Rules Inspector: by Automattic
Daniel Bachhuber – 1.3.1

Say What?: by Ademti Software – 2.1.2
Search Exclude: by Roman Pronskiy – 1.3.0
Simple Custom Post Order: by Colorlib – 2.5.6
Smush: by WPMU DEV – 3.12.4
Stream: by XWP – 3.9.1
TablePress: by Tobias Bäthge – 1.14
Term Management Tools: by theMikeD
scribu – 2.0.1

Transients Manager: by WPBeginner – 2.0.3
User Switching: by John Blackbourn & contributors – 1.7.0
WC Pickup Store: by Keylor Mendoza A. – 1.8.4
Woo Additional Terms: by MyPreview – 1.3.6
WooCommerce Email Test: by RaiserWeb – 1.2.2
Woocommerce Partial Shipment: by WpExperts Hub – 2.9
WooCommerce PayPal Checkout Gateway: by WooCommerce – 2.1.3
WooCommerce PayPal Payments: by WooCommerce – 2.0.0
WooCommerce Points and Rewards: by WooCommerce – 1.7.21
WooCommerce Smart Coupons: by StoreApps – 6.7.0
WooCommerce Stripe Gateway: by WooCommerce – 7.0.1
WooCommerce Subscriptions: by WooCommerce – 4.6.0
Woo Discount Rules: by Flycart – 2.5.2
Woo Discount Rules PRO 2.0: by Flycart – 2.5.2
Wordfence Security: by Wordfence – 7.8.0
WP All Export - ACF Export Add-On Pro: by Soflyy – 1.0.4
WP All Export - User Export Add-On Pro: by Soflyy – 1.0.7
WP All Export - WooCommerce Export Add-On Pro: by Soflyy – 1.0.5
WP All Export Pro: by Soflyy – 1.8.1
WP All Import - ACF Add-On: by Soflyy – 3.3.8
WP All Import - WooCommerce Import Add-On Pro: by Soflyy – 3.3.4
WP All Import Pro: by Soflyy – 4.7.8
WPCode Lite: by WPCode – 2.0.4.4
WP Data Access: by Passionate Programmers B.V. – 5.2.9
WPFront User Role Editor: by Syam Mohan – 3.2.1.11184
WP Mail Logging: by Wysija – 1.10.4
WP Mail SMTP: by WPForms – 3.6.1
WP phpMyAdmin: by Puvox.software – 5.2.0.7
WP Rocket: by WP Media – 3.12.3.2
Yoast SEO: by Team Yoast – 19.10

### Dropin Plugins (1) ###

advanced-cache.php: advanced-cache.php

### Must Use Plugins (1) ###

Disable Plugins: by KAGG Design – 1.2

### Settings ###

API Enabled: –
Force SSL: –
Currency: HKD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 0
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔
Enforce Approved Product Download Directories: –

### WC Pages ###

Shop base: #59181 - /the-shop/
Cart: #674 - /cart/
Checkout: #675 - /checkout/
My account: #676 - /my-account/
Terms and conditions: #513 - /terms-conditions/

### Theme ###

Name: Twenty Twenty-Two
Version: 1.3
Author URL: https://wordpress.org/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Admin ###

Enabled Features: activity-panels
analytics
coupons
customer-effort-score-tracks
experimental-products-task
experimental-import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
multichannel-marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: minified-js
new-product-management-experience
settings

Daily Cron: ✔ Next scheduled: 2022-11-30 05:12:55 +08:00
Options: ✔
Notes: 50
Onboarding: completed

### Action Scheduler ###

Complete: 45,976
Oldest: 2022-10-29 08:09:38 +0000
Newest: 2022-11-29 23:42:30 +0000

Pending: 5
Oldest: 2022-11-29 23:46:15 +0000
Newest: 2022-12-01 04:00:00 +0000

Failed: 75
Oldest: 2021-05-19 02:15:53 +0000
Newest: 2022-11-28 22:22:21 +0000

### Status report information ###

Generated at: 2022-11-30 07:46:40 +08:00
`

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by 1000camels** [comment]
Upon further investigation,/wp-includes/css/dist/components/style.css does not seem to be loaded.

**Comment by jonathansadowski** [comment]
Hi @1000camels,

Could you please provide a screenshot of how the arrows appear to you, so that we can compare and try to reproduce?

**Comment by 1000camels** [comment]
![image](https://user-images.githubusercontent.com/689727/204923005-0b5afc75-2dc2-4e44-9cf8-84bf45b9ff89.png)

**Comment by jjcoffee** [comment]
Also was able to replicate this on a fresh install. 

`/wp-includes/css/dist/components/style.css` does actually get loaded, the issue seems to be a missing react-dates dependency, probably because react-dates was [removed](https://github.com/WordPress/gutenberg/pull/43005) as of [Gutenberg 14.0](https://make.wordpress.org/core/2022/09/01/whats-new-in-gutenberg-14-0-31-august/). Enqueueing the CSS for react-dates on its own sorts the issue, e.g.

```
add_action( 'admin_enqueue_scripts', function(){
	wp_enqueue_style( 'react-dates', 'https://cdn.jsdelivr.net/npm/react-dates@21.8.0/lib/css/_datepicker.css' );
});
```

**Comment by swatipawarGS** [comment]
Hi @1000camels,

Thank you for taking the time to report this bug, we really appreciate your help. We are able to reproduce the issue on our end using the  WooCommerce 7.1.0 version and WordPress 6.1.1 version. 

Please find below the screenshot for reference :
![#35776](https://user-images.githubusercontent.com/94531721/205233894-099ea68b-50aa-4c22-ab12-ec7324a7e703.jpg)


Priority of this issue will be assigned shortly after we’d go through a round of prioritization. Please note it may take a couple of days.

**Comment by kaushikasomaiya** [comment]
Closing this, it would be fixed in an upcoming release: #35649
